src/utils/api.js

```diff
@@ -0,0 +1,44 @@
+const API_URL = "https://bootcamp-api.codeit.kr/api";
+
+async function getUserFolder() {
+  try {
+    const response = await fetch(`${API_URL}/sample/folder`, {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+
+    if (response.status !== 200) {
+      throw new Error(response.status);
+    }
+
+    const result = await response.json();
+    const folder = result.folder;
```

Comment:
다른 곳에서 잘 사용해주신 것처럼 구조분해할당을 사용해주시면 좋을 것 같습니다.

```suggestion
    const { folder } = result
```